src/main/scala/xiangshan/mem/lsqueue/LoadMisalignBuffer.scala

```diff
@@ -543,11 +552,18 @@ class LoadMisalignBuffer(implicit p: Parameters) extends XSModule
 
   io.writeBack.valid := req_valid && (bufferState === s_wb)
   io.writeBack.bits.uop := req.uop
-  io.writeBack.bits.uop.exceptionVec := Mux(
-    globalMMIO || globalException,
-    splitLoadResp(curPtr).uop.exceptionVec,
-    0.U.asTypeOf(ExceptionVec()) // TODO: is this ok?
-  )
+  io.writeBack.bits.uop.exceptionVec := 0.U.asTypeOf(ExceptionVec()) // FIXME: is this ok?
+  io.writeBack.bits.uop.exceptionVec(loadAddrMisaligned) :=
+    (globalMMIO || globalException) && exceptionVec(loadAddrMisaligned)
+  io.writeBack.bits.uop.exceptionVec(loadAccessFault) :=
+    (globalMMIO || globalException) && exceptionVec(loadAccessFault)
+  io.writeBack.bits.uop.exceptionVec(loadPageFault) :=
+    (globalMMIO || globalException) && exceptionVec(loadPageFault)
+  io.writeBack.bits.uop.exceptionVec(loadGuestPageFault) :=
+    (globalMMIO || globalException) && exceptionVec(loadGuestPageFault)
+  io.writeBack.bits.uop.exceptionVec(breakPoint) :=
+    (globalMMIO || globalException) && exceptionVec(breakPoint)
```

Comment:
`io.writeBack.bits.uop.exceptionVec := VecInit(exceptionVec.map(e => e && (...)))`

Reply:
<img width="745" alt="image" src="https://github.com/user-attachments/assets/1bc5c4b1-d274-4de9-9a3c-b927738ef41e">

```suggestion
  io.writeBack.bits.uop.exceptionVec := Mux(
    globalMMIO || globalException,
    splitLoadResp(curPtr).uop.exceptionVec,
    0.U.asTypeOf(ExceptionVec()) // TODO: is this ok?
  )
```

old assignment will generate redundant exceptions

Reply:
rewrite exceptionVec assign logic